tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "strict": true,
-    "strictNullChecks": true,
+    "strictNullChecks": false,
```

Comment:
any specific reason for disabling this?

Reply:
For any logic using `querySelector` or any other DOM element function, it's a nightmare to do the null checking. I think it's safe for us to leave this one off for now and turn it on in the future.

Reply:
Because we are doing one file at a time, I think is better to enforce it since the beginning the querySelector is going to return a value almost always, but can happens that it's not returning anything, I think we can handle that from the beginning, and my concern is about other null values in other situations

Reply:
Okay, I've turned it back on. We'll need to make sure we review pull requests very carefully to ensure null checks are being done properly.

In a lot of cases, the `!.` operator will be very useful.